README.md

```diff
@@ -15,7 +15,7 @@ This repository contains the source code of the demo application shown in the fo
 
 This sample application shows how to use the following libraries and features:
 
-- [Jetpack Compose Developer Preview 2](https://developer.android.com/jetpack/compose)
+- [Jetpack Compose Developer Preview 15](https://developer.android.com/jetpack/compose)
```

Comment:
There's no Developer Preview 15, there is `0.1.0-dev15` however. Names are confusing :)